e2e/js_image/BUILD.bazel

```diff
@@ -0,0 +1,58 @@
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@aspect_rules_js//js:defs.bzl", "js_binary")
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
+load("@io_bazel_rules_docker//container:layer.bzl", "container_layer")
+load(":js_image_layer.bzl", "js_image_layer")
+
+npm_link_all_packages(name = "node_modules")
+
+js_binary(
+    name = "main",
+    args = ["foo"],
+    data = [
+        "src/ascii.art",
+        "//:node_modules/chalk",
+    ],
+    entry_point = "src/main.js",
+)
+
+# Image
+js_image_layer(
```

Comment:
Nit: more commenting would be nice about what the resulting image looks like, and what node interpreter it gets